pkg/proto/configuration/http/http.proto

```diff
@@ -177,3 +185,12 @@ message AcceptHeaderAuthenticationPolicy {
   // provided.
   AuthenticationPolicy policy = 2;
 }
+
+message RemoteHttpRequestAuthenticationPolicy {
+  // The remote authenticator to grant or deny access the HTTP request.
+  buildbarn.configuration.grpc.RemoteAuthenticationPolicy backend = 1;
+
+  // Headers from the incoming HTTP request to forward to the remote
+  // authenticator.
+  repeated string headers = 2;
```

Comment:
Why isn't this field part of `RemoteAuthenticationPolicy`?

Reply:
In practive, it is the same. In the implementation, `headers` is used in a different layer, extracting the headers before going into the remote authorization. I suppose it's fine for the configuration code to split the structure and pass some variables to one constructor and other variables to another constructor. Better to have simplified configuration.